dali/pipeline/operators/reader/loader/video_loader.cc

```diff
@@ -439,16 +439,17 @@ void VideoLoader::PrepareEmpty(SequenceWrapper *tensor) {
 
 void VideoLoader::ReadSample(SequenceWrapper* tensor) {
     // TODO(spanev) remove the async between the 2 following methods?
-    push_sequence_to_read(filenames_[0], frame_starts_[current_frame_idx_], count_);
+    auto& fileidx_frame = frame_starts_[current_frame_idx_];
```

Comment:
I think we should provide ability to randomly select sample to read from shard for given GPU.